src/test/java/com/o19s/es/ltr/query/StoredLtrQueryBuilderTests.java

```diff
@@ -65,8 +66,9 @@
 public class StoredLtrQueryBuilderTests extends AbstractQueryTestCase<StoredLtrQueryBuilder> {
     private static final MemStore store = new MemStore();
 
+    // TODO: Remove the TestGeoShapeFieldMapperPlugin once upstream has completed the migration.
     protected Collection<Class<? extends Plugin>> getPlugins() {
-        return Collections.singletonList(TestPlugin.class);
+        return Arrays.asList(LtrQueryParserPlugin.class, TestGeoShapeFieldMapperPlugin.class);
```

Comment:
Here you need to keep the custom TestPlugin otherwize it'll fail with the message you reported about the call to the abstract method.